src/main/kotlin/game/racingcar/domain/RacingTrack.kt

```diff
@@ -0,0 +1,19 @@
+package game.racingcar.domain
+
+import game.racingcar.strategy.MoveStrategy
+
+class RacingTrack(private val cars: List<Car>) {
+    fun race(numberOfLap: Int, moveStrategy: MoveStrategy): List<Car> {
+        for (i in 0 until numberOfLap) {
+            cars.forEach { it.move(moveStrategy) }
+        }
+        return cars
+    }
+
+    fun getWinners(): List<Car> {
+        val maxPosition = cars.maxByOrNull { it.position }!!.position
```

Comment:
```suggestion
        val maxPosition = cars.maxOf { it.position }
```

`maxByOrNull`보다 `maxOf`를 사용하는 것이 더 명확하고 간결하지 않을까요?